benchmark/run.js

```diff
@@ -10,31 +10,54 @@ const cli = CLI(`usage: ./node run.js [options] [--] <category> ...
 
   --filter pattern          string to filter benchmark scripts
   --set    variable=value   set benchmark variable (can be repeated)
+  --format [simple|csv]     optional value that specifies the output format
 `, {
   arrayArgs: ['set']
 });
 const benchmarks = cli.benchmarks();
 
 if (benchmarks.length === 0) {
-  console.error('no benchmarks found');
-  process.exit(1);
+  console.error('No benchmarks found');
+  process.exitCode = 1;
+  return;
+}
+
+const validFormats = ['csv', 'simple'];
+const format = cli.optional.format || 'simple';
+if (!validFormats.includes(cli.optional.format)) {
```

Comment:
should be `!validFormats.includes(format)` otherwise it will print `Invalid format detected` when there are no `--format` argument.